server/src/main/java/org/apache/iotdb/db/query/executor/fill/LastPointReader.java

```diff
@@ -216,6 +216,9 @@ private boolean shouldUpdate(long time, long version, long newTime, long newVers
   }
 
   private boolean endtimeContainedByTimeFilter(Statistics statistics) {
+    if (timeFilter == null) {
+      return false;
+    }
```

Comment:
Same as before, if the filter is null, shoudn't we return true?

Reply:
Fixed